app/main.py

```diff
@@ -2,5 +2,15 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cached = {}
+
+    def inner(*args, **kwargs) -> Callable:
```

Comment:
Add pls type hint for `args` , and also you do not need here `kwargs` ( explained below ) . Also please think what type of data you returning , it is not Callable